doc/installation.rst

```diff
@@ -4,85 +4,94 @@ Installation
 Step 1: Install a `Miniconda <https://conda.io/miniconda.html>`_  (or `Anaconda <https://www.anaconda.com/downloads>`_) environment
 -----------------------------------------------------------------------------------------------------------------------------------
 
-Any Linux, Mac OS X, or Windows computer with a web browser (preferably Google Chrome) should be suitable. 16GB of RAM is required for some of the examples, but most will run fine in 4GB.
+Any Linux, Mac OS X, or Windows computer with a web browser (preferably
+Google Chrome) should be suitable. 16GB of RAM is required for some of
+the examples, but most will run fine in 4GB.
 
-If you don't already have conda on your machine, you can get it from `conda.io <https://conda.io/miniconda.html>`_, and then open a terminal window.
+If you don't already have conda on your machine, you can get it from
+`conda.io <https://conda.io/miniconda.html>`_, and then open a terminal
+window.
 
-If you do have conda already, it's a good idea to update it (running it twice to get the very latest) to ensure you have the latest version::
+If you do have conda already, it's a good idea to update it (running it
+twice to get the very latest) to ensure you have the latest version::
 
    > conda update conda
    > conda update conda
 
-You should then create a separate Conda environment to work in for this tutorial::
-
-   > conda create -n holoviz-tutorial python=3.7
-   > conda activate holoviz-tutorial
-
-Depending on how your conda is set up, you may need to use "source" instead of "conda" activate, or sometimes just "activate" on Windows.
-
-
-Step 2: Install ``holoviz`` and other required libraries
-------------------------------------------------------
+You should then make sure you have `anaconda-project` installed after
+activating your chosen conda environment:
 
 ::
 
-   > conda install -c pyviz holoviz
-
-If you have an **old version** run::
+   > conda install anaconda-project
 
-   > conda update -c pyviz holoviz
 
+Step 2: Download and extract the tutorial project
+-------------------------------------------------
 
-Step 3: Install the tutorials in your current working directory and download data
----------------------------------------------------------------------------------
+The following steps will fetch and extract the latest version of the
+tutorial project. *SciPy 2021 tutorial attendees should make sure to run
+these steps the day before the tutorial to ensure they have the latest
+version of the project.*
```

Comment:
Maybe "If you are attending a live tutorial or workshop, make sure to run these steps again the day before the tutorial to ensure  you have the same version of the project that the presenter will use.". Otherwise we'll have to update this page later!

Reply:
Applied suggestion in 2ae098f